<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SparseDialog</class>
 <widget class="QDialog" name="SparseDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2000</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>20</y>
     <width>231</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Rockwell</family>
     <pointsize>30</pointsize>
     <italic>true</italic>
     <bold>true</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>Sparse</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Digite Valor</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>110</y>
     <width>1541</width>
     <height>861</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTextEdit" name="txt_value">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>190</y>
     <width>104</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor" stdset="0">
    <cursorShape>IBeamCursor</cursorShape>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_insert">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>410</y>
     <width>120</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Rockwell</family>
     <pointsize>10</pointsize>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: rgb(124, 255, 144);
color: rgb(0, 0, 0);
</string>
   </property>
   <property name="text">
    <string>INSERTAR</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_regresarS">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>83</width>
     <height>50</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 85, 0);</string>
   </property>
   <property name="text">
    <string>Regresar</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="txt_value3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>310</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor" stdset="0">
    <cursorShape>IBeamCursor</cursorShape>
   </property>
  </widget>
  <widget class="QTextEdit" name="txt_value2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>250</y>
     <width>104</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor" stdset="0">
    <cursorShape>IBeamCursor</cursorShape>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
